packages/Header/Header.vue

```diff
@@ -1,14 +1,23 @@
 <template>
-  <div class="header">
-    <div class="container header-box">
-      <div class="title-box" :class="{ w50: !!image }">
-        <h1>{{ title }}</h1>
-        <slot>
-          <p>{{ description }}</p>
+  <div id="header">
+    <div v-if="image || $slots.image" class="container header-box">
+      <div class="title-box-image" half>
```

Comment:
I don't think this works, `half` is not present in the scss, it works only because the storyboard defines it

Reply:
`half` is present in the structure.scss file in the Theme. So do you think i should replace it by a class directly in the component?

Reply:
If we use the component without including the `structure.scss` then we will have an issue. 
The best solution would be to create components for the structure and like that we could use the component here. For now, let's say it's fine like that